tests/functional/testplan/runnable/interactive/interactive_executable.py

```diff
@@ -127,9 +127,9 @@
                 "type": "Fail",
                 "meta_type": "assertion",
                 "category": "DEFAULT",
-                "description": "Save a message into report",
+                "description": "None",
```

Comment:
So this is the last bit to fix. In `Fail` class the constructor falls back to message if description is missing, so it will not be `"None"`, but "`Save a message into report"` too.